frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -991,9 +1001,17 @@ private static function validateMemberOfReviewerGroup(\OmegaUp\Request $r) {
      *
      * @template T
      * @param array<T> $contents
+     * @return array{duplicates: string}
```

Comment:
Acá esta función está regresando un listado de tags duplicados, la declaración sería más bien así:
```suggestion
     * @return list<string>
```